mods/regedit-disable-beep.wh.cpp

```diff
@@ -0,0 +1,91 @@
+// ==WindhawkMod==
+// @id              regedit-disable-beep
+// @name            RegEdit Disable Beep
+// @description     Prevents RegEdit from beeping when an invalid key is entered to the navigation bar
+// @version         1.0.0
+// @author          ItsProfessional
+// @github          https://github.com/ItsProfessional
+// @include         regedit.exe
+// ==/WindhawkMod==
+
+// Source code is published under The GNU General Public License v3.0.
+
+// ==WindhawkModReadme==
+/*
+# RegEdit Auto Trim Whitespace on Navigation Bar
```

Comment:
You left the title from the previous mod.

Reply:
Fixed.